### PR TITLE
added 2 new tutorials

### PR DESCRIPTION
**Pull request by FidelVe**
added the following tutorials: 
* How to check if a validator is missing blocks
* How to create a wallet account

### CODE REVIEW

File: content/tutorials/2023-03-11-how-to-check-if-a-validator-is-missing-blocks/featured.jpg
Comment (han-so1omon) [inline content/tutorials/2023-03-11-how-to-check-if-a-validator-is-missing-blocks/featured.jpg:1]: Why is "IF" capitalized?

File: content/tutorials/2023-03-11-how-to-check-if-a-validator-is-missing-blocks/index.md [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+date: 2023-03-11
+author: espanicon
+slug: how-to-check-if-a-validator-is-missing-blocks
+description: Learn how to check if a validator is missing blocks consecutively.
+draft: false
+tags:
+- node-management
+- javascript
+---
+
```
Comment (han-so1omon) [inline content/tutorials/2023-03-11-how-to-check-if-a-validator-is-missing-blocks/index.md:12]: Please link to the full tutorial in the docs in the top or bottom of this tutorial

**Review by han-so1omon — CHANGES_REQUESTED** [review]

File: content/tutorials/2023-03-11-how-to-check-if-a-validator-is-missing-blocks/featured.jpg
Comment (FidelVe) [reply content/tutorials/2023-03-11-how-to-check-if-a-validator-is-missing-blocks/featured.jpg:1]: fixed

**Review by han-so1omon — APPROVED** [review]

**Comment by han-so1omon** [comment]
@DavidFBD looks good to me

**Pull request merged**